util/plancodec/codec_test.go

```diff
@@ -14,45 +14,45 @@
 package plancodec
 
 import (
-	. "github.com/pingcap/check"
 	"github.com/pingcap/tidb/kv"
+	"github.com/stretchr/testify/require"
+	"testing"
 )
```

Comment:
These imports violate goimports rule, you may setup an auto importer in GoLand (if you use it) as

![image](https://user-images.githubusercontent.com/18818196/126803358-ad9d1684-6fdb-40e3-b282-fe0f426128f6.png)